client/cli/src/params/shared_params.rs

```diff
@@ -43,6 +45,37 @@ pub struct SharedParams {
 	/// By default, all targets log `info`. The global log level can be set with -l<level>.
 	#[structopt(short = "l", long = "log", value_name = "LOG_PATTERN")]
 	pub log: Option<String>,
+
+	/// network address format
+	#[structopt(
+		long,
+		value_name = "NETWORK",
+		possible_values = &Ss58AddressFormat::all_names()[..],
+		parse(try_from_str = Ss58AddressFormat::try_from),
+		case_insensitive = true,
+		default_value = "polkadot"
+	)]
+	pub network: Ss58AddressFormat,
+
+	/// output format
+	#[structopt(
+		long,
+		value_name = "FORMAT",
+		possible_values = &OutputType::variants(),
+		case_insensitive = true,
+		default_value = "Text"
+	)]
+	pub output_type: OutputType,
+
+	/// cryptography scheme
+	#[structopt(
```

Comment:
These should not be part of `SharedParams`. How should I use `output_type` when running a normal node?

Reply:
true, good catch 👌🏾